docs/maps/ships.md

```diff
@@ -0,0 +1,43 @@
+<!-- replace _map_ with the actual map name -->
+<!-- change gamemode type for the Map data description  -->
+# Ships
+
+***
+
+#### ![shipsicon](../assets/maps/ships/ships-icon.jpg)
+
+# Overview
+***
+- **Introduced:** v1.7.0
+- **Description:** A battle between two boats over a lighthouse.
```

Comment:
Technically there are 4 ships. Thoughts on changing "boats" to "pirate ships"? Could update this to something like "A battle between pirates with ships on the shore."